packages/@vuepress/theme-default/util/index.js

```diff
@@ -233,13 +233,10 @@ function resolveItem (item, pages, base, groupDepth = 1) {
         title: item.title
       })
     }
-    return {
+    return Object.assign(item, {
       type: 'group',
-      path: item.path,
-      title: item.title,
-      sidebarDepth: item.sidebarDepth,
       children: children.map(child => resolveItem(child, pages, base, groupDepth + 1)),
```

Comment:
It's not recommended to delete these three lines. or we'll be hard to know which fields under `item` we really use.

Reply:
I am still not getting your point sir.

You meen this? 

```js
  return Object.assign(item, {
      type: 'group',
      path: item.path,
      title: item.title,
      sidebarDepth: item.sidebarDepth,
      children: children.map(child => resolveItem(child, pages, base, groupDepth + 1)),
      collapsable: item.collapsable !== false
    }
   })
```

But I still think there is no difference between above and below

```js
return Object.assign(item, {
      type: 'group',
      children: children.map(child => resolveItem(child, pages, base, groupDepth + 1)),
      collapsable: item.collapsable !== false
    }
 })
```

The path and the title are not `array` or `object`, they are all `string`.  What are the side effect?